framework/src/loops/ComputeUserObjectsThread.C

```diff
@@ -214,19 +228,37 @@ ComputeUserObjectsThread::onInternalSide(const Elem * elem, unsigned int side)
   for (const auto & uo : _internal_side_objs)
     if (!uo->blockRestricted() || uo->hasBlocks(neighbor->subdomain_id()))
       uo->execute();
+
+  for (auto & uo : _domain_objs)
+    if (!uo->blockRestricted() || uo->hasBlocks(neighbor->subdomain_id()))
+    {
+      uo->preExecuteOnInternalSide();
+      uo->executeOnInternalSide();
+    }
 }
 
 void
 ComputeUserObjectsThread::onInterface(const Elem * elem, unsigned int side, BoundaryID bnd_id)
 {
   // Pointer to the neighbor we are currently working on.
   const Elem * neighbor = elem->neighbor_ptr(side);
+  if (!(neighbor->active()))
+    return;
 
   std::vector<UserObject *> userobjs;
```

Comment:
I think this variable name is confusing now. 
should be interface_uo or contain both InterfaceUO and domainUO